### DISCUSSION TITLE
Device Group Filter

### DISCUSSION THREAD

**Discussion by DaanSelen** [Ylianst/MeshCentral#6349] [Ideas]
Hello everyone, I wanted to ask if its possible to filter for device groups instead of devices themselves?

**Comment by si458** [comment]
Plz read the docs https://ylianst.github.io/MeshCentral/meshcentral/devicetabs/#search-or-filter

I know the docs aren't complete but information is in there!

We can always do with the help updating/adding things to the docs here https://github.com/Ylianst/MeshCentral/tree/master/docs/docs

**Reply by DaanSelen** [reply]
Excuse me Simon, sorry! But true its there!
